README.rst

```diff
@@ -227,6 +227,20 @@ Installing the latest master branch of Salt:
   curl -L https://bootstrap.saltstack.com | sudo sh -s -- git master
 
 
+Install on Windows
+~~~~~~~~~~~~~~~~~~
+
+Using ``curl.exe`` to install latest development version from GitHub:
+
+.. code:: console
+
+  curl.exe -o bootstrap-salt.ps1 -L 
+  https://raw.githubusercontent.com/saltstack/salt-bootstrap/develop/bootstrap-salt.ps1
```

Comment:
https://winbootstrap.saltstack.com is what should be used.

Reply:
And there is no `curl` on Windows by default. You should probably give some powershell code here:
```
Invoke-WebRequest `
    -Uri https://raw.githubusercontent.com/saltstack/salt-bootstrap/develop/bootstrap-salt.ps1 `
    -OutFile C:\Temp\bootstrap-salt.ps1
```

Reply:
Thanks @bryceml   that is better.

Reply:
Good point @twangboy .. I was using cygwin which includes curl.  
I added a second example. comments welcome as I'm not familiar with PS.